CondCore/ESSources/plugins/CondDBESSource.h

```diff
@@ -120,6 +120,8 @@ class CondDBESSource : public edm::eventsetup::ESProductResolverProvider, public
   typedef std::map<std::string, cond::GTEntry_t> TagCollection;
   // the collections of tag, record/label used in this ESSource
   TagCollection m_tagCollection;
+  std::vector<std::string> m_recordsToDebug;
```

Comment:
I'm considering changing it to recordAndLabelsToDebug and eg. let the user pass a vector of strings of records concatenated with labels separated by "@" (eg. `"MyRecord1Rcd@Label1", "MyRecord1Rcd@Label2", "MyRecord2Rcd"`)

Do you think it would be worth implementing in this PR? Maybe you have an idea of how to do it more intuitively for the user?

Reply:
I think we can leave it as it is for now and if there's a need for more precise choice we can implement it then